src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/profile/Profile.java

```diff
@@ -0,0 +1,387 @@
+package io.github.oliviercailloux.y2018.apartments.valuefunction.profile;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.collect.BoundType;
+import com.google.common.collect.Range;
+import io.github.oliviercailloux.y2018.apartments.valuefunction.Criterion;
+import io.github.oliviercailloux.y2018.apartments.valuefunction.LinearAVF;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Profile {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(Profile.class);
+
+  private LinearAVF linearAvf;
+
+  /**
+   * The 10 next arguments gives the Range of apartment characteristic subjective value weight in
+   * the calculation of the Apartment total subjective value
+   */
+  private Range<Double> floorAreaWeightRange;
+
+  private Range<Double> nbBedroomsWeightRange;
+  private Range<Double> nbSleepingWeightRange;
+  private Range<Double> nbBathroomsWeightRange;
+  private Range<Double> terraceWeightRange;
+  private Range<Double> floorAreaTerraceWeightRange;
+  private Range<Double> wifiSubjectiveValueWeight;
```

Comment:
Manque Range dans le nom des variables pour bien les différencier de la valeur unique du poids.

Reply:
Fixed in commit 4983c6c